lib/XMLHttpRequest.js

```diff
@@ -504,7 +538,7 @@ exports.XMLHttpRequest = function () {
    */
   this.handleError = function (error) {
     this.status = 0
-    this.statusText = error
```

Comment:
(this belongs in the previous review) An error is not a string, so stringify it.  For some reason the sync and async errors have slightly different info, but for the cert test they both have enough in common to recognize.